### ISSUE TITLE
ionic-angular:  NX   Unable to resolve @nxext/ionic-angular:application

### ISSUE THREAD

**Issue by rbalet** [nxext/nx-extensions#1118]
**Describe the bug**
I followed [the documentation](https://nxext.github.io/nx-extensions-ionic/docs/ionic-angular/getting-started.html) and ran `nx generate @nxext/ionic-angular:application myApp`. 
I do get `NX   Unable to resolve @nxext/ionic-angular:application.`

**To Reproduce**
1. `npm install --save-dev --exact @nxext/ionic-angular`
2. `nx generate @nxext/ionic-angular:application myApp`

**Expected behavior**
Generating the app

**Additional context**
```
    "@nx/angular": "19.8.4",
    "@nx/devkit": "19.8.4",
    "@nx/eslint": "19.8.4",
    "@nx/eslint-plugin": "19.8.4",
    "@nx/jest": "19.8.4",
    "@nx/js": "19.8.4",
    "@nx/playwright": "19.8.4",
    "@nx/web": "19.8.4",
    "@nx/workspace": "19.8.4",
    "@nxext/capacitor": "19.1.0",
    "@nxext/ionic-angular": "^19.1.0",
```

**Comment by edbzn** [comment]
Hey, please refer to https://nxext.dev/docs/ionic-angular/generators.html for the docs, it's incomplete and imperfect but up to date. The docs https://nxext.github.io is outdated and should be turned off (cc @DominikPieper). 
FYI there's no `application` generator, it was replaced by a `configuration` generator that takes an existing app. Note also that we just released `@nxext/ionic-angular@19.1.1` with various fixes.

**Comment by DominikPieper** [comment]
@edbzn the interesting thing here is, github pages is disabled since some time ^^' I'll take a look how to remove this one
